ipn/backend.go

```diff
@@ -92,6 +93,48 @@ type Notify struct {
 	// type is mirrored in xcode/Shared/IPN.swift
 }
 
+func (n Notify) String() string {
```

Comment:
I'd make this a pointer receiver.

Reply:
It turns out that doesn't work, since non-pointer Notify objects are passed around. (Maybe that in itself is the mistake.) If you try to fmt.Sprintf("%v", n) where n is not a pointer, Go doesn't find the pointer version of String. (This has always seemed weird to me, maybe I'm missing something.)